src/pages/jonghyeoklim/Login/LoginJongHyeok.scss

```
@@ -0,0 +1,95 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  background-color: #fafafa;
+}
+
+.container {
+  width: 500px;
+  height: 550px;
+  margin: 0 auto;
+  margin-top: 50px;
+  border: 1px solid #e8e8e8;
+  background-color: white;
+}
+
+.logo_name {
+  display: flex;
+  justify-content: center;
+  margin: 30px 0;
+}
+
+.name {
+  font-size: 60px;
+  margin-left: auto;
+  margin-right: auto;
```

잘 모르지만 margin left와 right가 auto라면 합칠수 있지않을까요??
아니면 죄송합니다..

---

해볼게요~~

---

top, bottom 에 영향이 없다면 단축 속성으로 활용
```suggestion
  margin: 0 auto;
```